#!/bin/sh /usr/share/dpatch/dpatch-run
## 998-doc-manpages.dpatch by Michael Piefel <piefel@debian.org>
##
## DP: FIXME, and #417511.

@DPATCH@

diff -Naurp file-4.24.orig/doc/file.man file-4.24/doc/file.man
--- file-4.24.orig/doc/file.man	2008-03-07 15:00:07.000000000 +0000
+++ file-4.24/doc/file.man	2008-04-07 08:26:06.000000000 +0000
@@ -51,7 +51,7 @@ meaning anything else (data is usually 
 or non-printable).
 Exceptions are well-known file formats (core files, tar archives)
 that are known to contain binary data.
-When modifying magic files or the program itself, make sure to
+When adding local definitions to /etc/magic, make sure to
 .Em "preserve these keywords" .
 Users depend on knowing that all the readable files in a directory
 have the word 
@@ -95,7 +95,8 @@ The concept of a
 has been applied by extension to data files.
 Any file with some invariant identifier at a small fixed
 offset into the file can usually be described in this way.
-The information identifying these files is read from the compiled
+The information identifying these files is read from /etc/magic
+and the the compiled
 magic file
 .Pa __MAGIC__.mgc ,
 or the files in the directory
@@ -326,6 +327,11 @@ will not attempt to open
 adds
 .Dq .mgc
 to the value of this variable as appropriate.
+However,
+.Pa file
+has to exist in order for
+.Pa file.mime
+to be considered.
 The environment variable
 .Dv POSIXLY_CORRECT
 controls (on systems that support symbolic links), whether
@@ -401,12 +407,6 @@ will be distributed periodically.
 The order of entries in the magic file is significant.
 Depending on what system you are using, the order that
 they are put together may be incorrect.
-If your old
-.Nm
-command uses a magic file,
-keep the old magic file around for comparison purposes
-(rename it to 
-.Pa __MAGIC__.orig ).
 .Sh EXAMPLES
 .Bd -literal -offset indent 
 $ file file.c file /dev/{wd0a,hda}
@@ -548,3 +548,8 @@ on
 .Dv ftp.astron.com
 in the directory
 .Dv /pub/file/file-X.YZ.tar.gz
+.Pp
+This Debian version adds a number of new magix entries. It can be
+obtained from every site carrying a Debian distribution (that is
+.Dv ftp.debian.org
+and mirrors).
